### DISCUSSION TITLE
how to get a R Variable Queries

### DISCUSSION THREAD

**Discussion by morning-start** [microsoft/vscode-jupyter#9441] [Questions and Answers] [answered]
How I set, could get a Variable Queries for R?

```
  "jupyter.variableQueries": [
    {
      "language": "R",
      "query": "%who_ls",
      "parseExpr": "'(\\w+)'"
    }

```

this is doesn't work.

**Comment by DonJayamanne** [comment, marked as answer]
Thanks for your comments, & I'm sorry it doesn't work. 
Please could you update this issue https://github.com/microsoft/vscode-jupyter/issues/5264 with the following information:
* Please provide the output from the `Jupyter` output panel

**Reply by DonJayamanne** [reply]
I'm going to close this discussion in favor of the issue https://github.com/microsoft/vscode-jupyter/issues/5264
